text-inputs/text-inputs/src/main/java/com/decathlon/vitamin/compose/textinputs/VitaminTextInputs.kt

```diff
@@ -405,7 +351,10 @@ internal fun VitaminTextInputLayoutImpl(
             textInput()
         }
         Row(
-            modifier = Modifier.padding(vertical = 1.dp, horizontal = 4.dp)
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(vertical = 1.dp, horizontal = 4.dp),
+            horizontalArrangement = Arrangement.End,
```

Comment:
Why the arrangment.End is need there ?

Reply:
I need my counter to wrap its content but always on the right(end) even if no helperText. 
The helperText will take all the space left